src/sentry/migrations/0164_my_issues_saved_search.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 1.11.29 on 2021-02-19 00:08
+
+from django.db import migrations
+
+
+def add_my_issues_search(apps, schema_editor):
+    SavedSearch = apps.get_model("sentry", "SavedSearch")
+    search = SavedSearch.objects.create(
+        name="My Issues",
+        query="is:unresolved assigned_or_suggested:me",
+        organization_id=None,
+        is_default=False,
+        is_global=True,
+        # models.search_common.SearchType.ISSUE
+        type=0,
+        sort="date",
+    )
+    search.save()
```

Comment:
No need for `.save()`, you already created it which will insert a row. I think this was just an oversight in the last migration that adds a global row.

Reply:
you have seen right through my copy and paste